playwright/src/main/java/com/microsoft/playwright/junit/Options.java

```diff
@@ -83,6 +84,15 @@ public Options setBrowserName(String browserName) {
     return this;
   }
 
+  public String getDeviceName() {
```

Comment:
It's unfortunate that we cannot read back the actual values here, only the device name. However, that's better than nothing.